jreleaser.yml

```diff
@@ -0,0 +1,46 @@
+project:
+  name: spoon-core
+  description: A plugin to lock and validate the integrity of the dependencies of a maven based project.
+  longDescription: Spoon is an open-source library to analyze, rewrite, transform, transpile Java source code. 
```

Comment:
Are you sure indentation in YAML works like this? I have no idea, but it looks a bit weird in github.

Reply:
The file with the same formatting works for maven lockfile. So, I'm confident it should work.